outOfSyncNoti.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
-# A simple shell script to check if node is out of sync with ethereum testnet
-# If node is out of sync, user will be nptofied by OpsGenie
-# Using crontab command: `*/10 * * * * /home/someuser/tmp/` to run this script every 10 mins.
+# A simple shell script to check if node is out of sync with ethereum
+# If node is out of sync, user will be notified by OpsGenie
+# Using crontab command: `*/10 * * * * /home/foo/outOfSyncNoti.sh 'rinkeby' 'opsgeniekeyhere' >> /home/foo/nodeSync.log 2>&1` to run this script every 10 mins.
+
+network=$1
+port=$2
+genieKey=$3
+threshold=${4:-200}
 
 # Get current node block number
-curBlock=$(eval geth --exec "eth.blockNumber" attach http://localhost:8545)
+curBlock=$(eval geth --exec "eth.blockNumber" attach http://localhost:$port)
+if [ "$?" = "1" ]; then
+  curBlock=0
+fi
 
 # Get latest block number from etherscan
 # If on mainnet
-if [ "$1" == "mainnet" ]; then
+if [ "$network" == "mainnet" ]; then
```

Comment:
the below 2 commands differ only in etherscan subdomain name: let's introduce variable like `etherscanDomain` and assign it depending on network in this `if` and then perform the command.